osu.Game/Scoring/ScoreManager.cs

```diff
@@ -55,9 +55,7 @@ protected override ScoreInfo CreateModel(ArchiveReader archive)
 
         public Score GetScore(ScoreInfo score) => new LegacyDatabasedScore(score, rulesets, beatmaps, Files.Store);
 
-        public List<ScoreInfo> GetAllUsableScores() => ModelStore.ConsumableItems.Where(s => !s.DeletePending).ToList();
-
-        public IEnumerable<ScoreInfo> QueryScores(Expression<Func<ScoreInfo, bool>> query) => ModelStore.ConsumableItems.AsNoTracking().Where(query);
+        public IEnumerable<ScoreInfo> GetAllUsableScores() => ModelStore.ConsumableItems.Where(s => !s.DeletePending);
```

Comment:
why this change?

Reply:
Couldyou not use `QueryScores` below? These methods generally should match on all manager classes (we probably want an interface for this going forward).

Reply:
Resolved in https://github.com/ppy/osu/pull/4000/commits/097062caf78bd7985fa4d1af9b675743191a6671